tests/Http/ClientTest.php

```diff
@@ -231,6 +231,29 @@ public function testClientHasDefaultUserAgent(): void
         $this->assertTrue($client->isHealthy());
     }
 
+    public function testEmptyMasterkeyRemovesAuthHeader(): void
+    {
+        $httpClient = $this->createHttpClientMock(200, '{}');
+        $reqFactory = $this->createMock(RequestFactoryInterface::class);
+        $requestStub = $this->createMock(RequestInterface::class);
+        $requestStub->expects($this->once())
+            ->method('withAddedHeader')
+            ->willReturnCallback(function ($name, $value) use ($requestStub) {
+                if ('Authorization' === $name) {
+                    $this->assertSame('Bearer ', $value);
+                }
```

Comment:
this actually tests the wrong thing. I'll update the test now

Reply:
@norkunas thanks for the suggestion, didn't see the comment while providing a fix 😅 didn't write PHP tests in a while.

Can you check the latest commit?

https://github.com/meilisearch/meilisearch-php/pull/585/commits/dd2d9db35bc5c42d9a298f84ef884f341488a843

Reply:
looks good :)

Reply:
on the second thought, as there is `$this->once()`, the else case will never be reached I think.
so this would be enough:
```
        $requestStub->expects($this->once())
            ->method('withAddedHeader')
            ->willReturnCallback(function ($name, $value) use ($requestStub) {
                $this->assertSame('User-Agent', $name);
                $this->assertSame(Meilisearch::qualifiedVersion(), $value);
```